stdeb.cfg

```diff
@@ -1,6 +1,6 @@
 [DEFAULT]
 Package3: onionshare
-Depends3: python3-flask, python3-stem, python3-pyqt5, python-nautilus, tor, obfs4proxy
-Build-Depends: python3-pytest, python3-flask, python3-stem, python3-pyqt5
+Depends3: python3-flask, python3-stem, python3-pyqt5, python3-cryptography, python3-crypto, python3-nacl, python3-socks, python-nautilus, tor, obfs4proxy
+Build-Depends: python3-pytest, python3-flask, python3-stem, python3-pyqt5, python3-cryptography, python3-crypto, python3-nacl, python3-socks, python-nautilus, tor, obfs4proxy
 Suite: bionic
 X-Python3-Version: >= 3.6
```

Comment:
Does `X-Python3-Version: >= 3.6` mean it is literally *impossible* to build a .deb on a platform that has an older Python3 version?

Reply:
Related to this and the above comment - is there any implication for Tails here, if they decide to package the latest OnionShare? I thought 3.x was also based on Debian Stretch (Python 3.5).

I feel like adding python3-sha3 might be important for such projects in case they decide to package downstream.. though I appreciate it looks not possible for Fedora..